src/routes/safe/components/Transactions/TxList/hooks/useKnownAddress.ts

```diff
@@ -0,0 +1,18 @@
+import { useSelector } from 'react-redux'
+
+import { getNameFromAddressBookSelector } from 'src/logic/addressBook/store/selectors'
+
+type AddressInfo = { name: string | undefined; image: string | undefined }
+
+export const useKnownAddress = (address: string, addressInfo: AddressInfo): AddressInfo => {
+  const recipientName = useSelector((state) => getNameFromAddressBookSelector(state, address))
+
+  const isInAddressBook = recipientName !== 'UNKNOWN'
+
+  return isInAddressBook
+    ? {
+        name: recipientName,
+        image: undefined,
```

Comment:
Why is the image undefined in this case?
Can it be the case that `addressInfo` has the image?

Reply:
no, because when the address is in the address book, we want to render de Identicon.